cpp/daal/src/algorithms/k_nearest_neighbors/bf_knn_classification_model_impl.cpp

```diff
@@ -64,7 +64,8 @@ size_t Model::getNumberOfFeatures() const
 
 services::Status Parameter::check() const
 {
-    DAAL_CHECK_EX(this->nClasses > 0, services::ErrorIncorrectParameter, services::ParameterName, nClassesStr());
+    DAAL_CHECK_EX(this->nClasses > 1, services::ErrorIncorrectParameter, services::ParameterName, nClassesStr());
```

Comment:
it is necessary to add badarg tests for these parameters. we can't work with one class for this algorithm?

Reply:
badarg added, nClasses = 1 is non default and obviously incorrect if set by user.